toolchain/check/testdata/array/generic_empty.carbon

```diff
@@ -23,6 +23,7 @@ fn G(T:! type) {
 // CHECK:STDOUT:   %G: %G.type = struct_value () [template]
 // CHECK:STDOUT:   %int_0: Core.IntLiteral = int_value 0 [template]
 // CHECK:STDOUT:   %array_type: type = array_type %int_0, %T [symbolic]
+// CHECK:STDOUT:   %.1: <witness> = require_complete_type %array_type [symbolic]
```

Comment:
Can `require_complete_type` forward the name of its argument, e.g. `%array_type.complete`? The `%.1` just feels like it may increase churn.